keras/callbacks.py

```diff
@@ -1946,6 +1946,10 @@ class EarlyStopping(Callback):
           of the performance relative to the `baseline`. If no epoch
           improves on `baseline`, training will run for `patience`
           epochs and restore weights from the best epoch in that set.
+      start_from_epoch: Number of epochs to wait before starting
+          to monitor improvement. This allows a warm-up period in which
```

Comment:
"This allows for"

Reply:
Thanks @fchollet - much appreciated! of-course I fixed.